src/highlight/HighlightAnnotations.tsx

```diff
@@ -97,6 +106,8 @@ const HighlightAnnotations = (props: Props): JSX.Element => {
                     />
                 </div>
             )}
+
+            {!isCreating && selection && <PopupHighlight onClick={handlePromote} rect={selection.boundingRect} />}
```

Comment:
I suggest wrapping this in a div with a class of `ba-HighlightAnnotations-popup` so you can control all of the pointer events styles in one place. It would also be more consistent with region annotations.